lib/modules/scroller.js

```diff
@@ -212,7 +211,11 @@ const selector = createSelector(
 );
 
 const dispatcher = (dispatch, ownProps) => ({
-  onSaveScrollPosition: scrollPosition => ownProps.id && dispatch(saveScrollPosition(ownProps.id, scrollPosition)),
```

Comment:
lastly, this rule as well :) there are a few places in our templates where we've been using `foo && <div/>` to conditionally render. we should formalize what direction we want to go before we make this change.